.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "src/SOILWAT2"]
 	path = src/SOILWAT2
 	url = https://github.com/DrylandEcology/SOILWAT2
-	branch = master
+	branch = feature_reentrancy_threadsafety
```

Comment:
please update SOILWAT2 module to point to branch release/devel_v7.1.0 before we merge this